samply/share-client
# Rename future builds from samply-* to gbn-*

**What's in the PR**
* Docker Images build by ci.yml will now be named gbn-* instead of samply-*.

**How to test manually**
* Visit [Docker Hub](https://hub.docker.com/r/samply/share-client/tags?page=1&ordering=last_updated) and search for gbn-* images. After sucessful build where should be an image gbn-renameSamplyToGBN.
